package.json

```diff
@@ -54,6 +54,9 @@
     "tailwindcss": "^3.0.7"
   },
   "dependencies": {
-    "open-graph-scraper": "^4.11.0"
+    "open-graph-scraper": "^4.11.0",
+    "prop-types": "^15.7.2",
```

Comment:
I don't think proptypes are providing any value in the project. The react community has moved to TypeScript, but I would also not recommend that at this time either.

Reply:
I had to introduce propTypes since eslint screams at props whose Types are not defined.

The work around would then be to tell eslint not to scream when props are not defined? 😨